src/view-components/core-views.tsx

```diff
@@ -23,12 +23,12 @@ export const coreWidgetDefinitions: WidgetConfig[] = [
 
   {
     name: "MedicationsOverview",
-    esModule: "@openmrs/esm-patient-chart-widgets"
+    extensionSlotName: "patient-chart-dashboard-medications"
```

Comment:
I'm not sure whether we want to include this already because nothing will be displayed until the drugorder widget is deployed. We can also remove/delay these changes here until that happened.
What do you think? 

@FlorianRappl

Reply:
I'd say lets keep it. We'll bring that online soon!